webrender/src/renderer.rs

```diff
@@ -1878,6 +1904,22 @@ impl Renderer {
                                       &target.text_run_textures,
                                       &projection);
         }
+        if !target.line_cache_prims.is_empty() {
+            // TODO(gw): Technically, we don't need blend for solid
+            //           lines. We could check that here?
```

Comment:
if that becomes a performance concern, we'd need to split those into opaque/non-opaque